homeassistant/components/litterrobot/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
```

Comment:
The RequiredKeyMixins can be removed as they can now be combined with the normal extended entity description if you add `kw_only=True` to the dataclass annotation